nixos/modules/services/networking/tailscale.nix

```diff
@@ -58,6 +58,13 @@ in {
         A file containing the auth key.
       '';
     };
+
+    extraUpFlags = mkOption {
+      description = lib.mdDoc "Extra flags to pass to {command}`tailscale up`.";
+      type = types.separatedString " ";
```

Comment:
please make this a `listOf str` and then use `escapeShellArgs` to turn it into one unified string.